### ISSUE TITLE
JAWS doesn't announce change of language for content inside button tags

### ISSUE THREAD

**Issue by brooksienoodlesoup** [FreedomScientific/standards-support#340]
### Summary

For button labels that are provided by the content that falls between the opening and closing button element tags, designated changes of language are not announced by JAWS using the default Eloquence voice.

### Expected result

I expected JAWS to announce the button label using two different accents - English for "Start /" and Spanish for "Iniciar."

### Actual result

JAWS read all of the button label in an English accent.

### Example

Example code:

```
<html lang="en">
..
<button>Start /<span lang="es"> Iniciar</span></button>
..
</html>
```

#### JAWS version and build number

2019.1909.28 ILM
 
#### Operating System and version

Windows 7

#### Browser and version:

Firefox 70.0.1

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/accname/issues/56

**Comment by JAWS-test** [comment]
The problem also occurs in Chrome and IE 11. In general, the `lang` attribute in JAWS only works reliably with texts, not with interactive elements. 

In the button example, a distinction must be made between tab navigation and arrow navigation. Is different from browser to browser, sometimes correct, but never correct for every navigation method.

It is interesting to note that in Firefox and IE 11 the output of the button depends on whether the language change takes place at the beginning or not at the beginning.

```
<button>Start /<span lang="es"> Iniciar</span></button>
```
is output with English pronunciation, but
```
<button><span lang="es"> Iniciar</span> / Start </button>
```
is output with Spanish pronunciation.

As long as the many problems with the output of language changes in JAWS are not fixed, language changes should be avoided as much as possible.

**Comment by dfarough** [comment]
another situation where this occurs is for field sets.
I notice that the legend of a fieldset will be spoken in English even though the lang is set to fr
try the following code.

<fieldset>

<legend lang='fr' class="simpleSmallerLegend">Plusieurs personnes auront la chance d’obtenir une chaise ergonomique _______ les mois à venir.</legend>
<input type="radio" disabled="disabled" lang='fr'  id="radio1"><label for="radio1">par</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio2"><label for="radio2">sur</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio3"><label for="radio3">en</label><br />
<input type="radio" disabled="disabled" lang='fr' checked="checked"  id="radio4"><label for="radio4">dans</label>

</fieldset>

**Comment by dfarough** [comment]
<fieldset>

<legend lang='fr' class="simpleSmallerLegend">Plusieurs personnes auront la chance d’obtenir une chaise ergonomique _______ les mois à venir.</legend>
<input type="radio" disabled="disabled" lang='fr'  id="radio1"><label for="radio1">par</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio2"><label for="radio2">sur</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio3"><label for="radio3">en</label><br />
<input type="radio" disabled="disabled" lang='fr' checked="checked"  id="radio4"><label for="radio4">dans</label>

</fieldset>

**Comment by dfarough** [comment]
`<fieldset>

<legend lang='fr' class="simpleSmallerLegend">Plusieurs personnes auront la chance d’obtenir une chaise ergonomique _______ les mois à venir.</legend>
<input type="radio" disabled="disabled" lang='fr'  id="radio1"><label for="radio1">par</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio2"><label for="radio2">sur</label><br />
<input type="radio" disabled="disabled" lang='fr'  id="radio3"><label for="radio3">en</label><br />
<input type="radio" disabled="disabled" lang='fr' checked="checked"  id="radio4"><label for="radio4">dans</label>

</fieldset>
`

**Comment by JAWS-test** [comment]
> another situation where this occurs is for field sets.

I have already reported this error in https://github.com/FreedomScientific/VFO-standards-support/issues/233 . Basically the following applies: The language change of interactive elements often does not work and should be avoided.

**Comment by JeremiahRogersVispero** [comment]
Bug 172730
